### DISCUSSION TITLE
Email Capability

### DISCUSSION THREAD

**Discussion by SterlingPeet** [nasa/fprime#721] [General]
My primary desire is to receive F Prime questions and answers in my email.  I think that this system does that, assuming I have the correct settings.  Perhaps someone could reply to this message to test it out?

PS: I do like that this format includes markdown composition!

Thanks,

Sterling Peet

**Comment by Joshua-Anderson** [comment]
Hi @SterlingPeet! Any luck with getting email working?

**Comment by SterlingPeet** [comment]
It looks like it works, this is coming from an email reply!

Thanks,

Sterling Peet

On Jun 15, 2021, at 9:02 PM, Joshua Anderson ***@***.******@***.***>> wrote:



Hi @SterlingPeet<https://github.com/SterlingPeet>! Any luck with getting email working?

**Comment by SterlingPeet** [comment]
OK, so I clearly get emails from the discussion that I comment on, but how do I subscribe to get emails from every discussion?

**Comment by saba-ja** [comment]
@SterlingPeet, I think you can either watch everything from this repo or select Custom watch and check the Discussion notification as shown below:

![Screen Shot 2021-06-17 at 11 47 59 AM](https://user-images.githubusercontent.com/35859004/122456087-fbe81380-cf61-11eb-912c-b0f22df04e1e.png)

**Reply by SterlingPeet** [reply]
Oh cool, thats exactly what I wanted!
